packages/patternfly-4/react-core/README.md

```diff
@@ -64,23 +64,25 @@ Note: All commands below assume you are on the root directory in this repository
 
 ### Install
 
-Run to install all the dependencies
+Run to install all the dependencies.
 
 ```sh
 yarn install && yarn build
 ```
 
 ### Running
 
-To start the site locally.
+To launch the development server and view the workspace:
 
 ```sh
 yarn start:pf4
 ```
 
+Go to localhost:8000.
+
```

Comment:
Although adding more info to the README is a good thing, this does not appear to address the original issue. That is, the `yarn start:dev` command no longer confirms exactly where the app is running.

This also does not account for the example app running on a different port, should port 8000 be occupied already.

Reply:
@dlabrecq Thank you for your feedback Dan! This has to do with something Zack addressed here: https://github.com/patternfly/patternfly-react/issues/3154#issuecomment-545600553 and I added that line based on what was agreed upon in this comment.

Good point about the different port -- do you happen to know what port the app runs on if localhost:8000 is already occupied?

Reply:
@jenny-s51 if the port is already in use it prints to the console `@patternfly/react-docs: Something is already running at port 8000`. I think it used to be the case that an interactive console would take over and allow you to specify a different port, then proceed to launch the dev server using that new config. It seems to have regressed or been removed at some point. 

I'm personally ok with pulling these changes in, but I agree it doesn't really fix the root cause so as long as we can leave the related issue open for future fixes we should be ok.

Reply:
Ok, can leave that issue opened